keras-dreambooth-sprint/README.md

```diff
@@ -1,95 +1,135 @@
-DreamBooth was proposed in [DreamBooth: Fine Tuning Text-to-Image Diffusion Models for Subject-Driven Generation](https://arxiv.org/abs/2208.12242) by Ruiz et al.
-In this guide, we will walk you through what we will do in this event.
+Welcome to Keras Dreambooth event!
 
-We will be training Dreambooth models using KerasCV and building demos on them. 
+TODO: Add some introduction here
 
-**Getting Started** 🚀 
+This document summarises all the relevant information required for the event 📋. Please read it thoroughly.
 
-To get started, join us in [hf.co/join/discord](http://hf.co/join/discord) and take the role #open-source, and meet us in #keras-working-group channel. 
+## Table of Contents
 
-We will be hosting our demos in this organization on Hugging Face Hub: [keras-dreambooth](https://huggingface.co/keras-dreambooth), send a request to join [here](https://huggingface.co/organizations/keras-dreambooth/share/RMocthadPgpxxUDHtAesrbBzieDLgUfPmv) if you’d like to make a submission 🙂
+* [Introduction](#introduction)
+* [Important Dates](#important-dates)
+* [Getting Started](#getting-started--)
+* [Submission Categories](#submission-categories)
+* [Compute](#compute)
+* [Model Training](#model-training)
+* [Model Hosting](#model-hosting)
+* [Building Demo](#building-demo)
+* [Sprint Prizes](#sprint-prizes)
 
-### But what are we going to do?
+## Introduction
 
-We will:
+Dreambooth is a fine-tuning technique to teach new visual concepts to text-conditioned Diffusion models with just 3-5 images. With Dreambooth, you could generate funny and realistic images of your dog, yourself and any concept with few images using Stable Diffusion. DreamBooth was proposed in [DreamBooth: Fine Tuning Text-to-Image Diffusion Models for Subject-Driven Generation](https://arxiv.org/abs/2208.12242) by Ruiz et al.
+
+We will be training Dreambooth models using KerasCV, sharing the models and building demos using the models.
 
-1. Fine-tune Stable Diffusion on any concept we want using Dreambooth,
-2. Push the model to Hugging Face Hub,
+We will:
+1. Fine-tune Stable Diffusion on your chosen concept using Dreambooth with KerasCV,
+2. Push the model to the Hugging Face Hub,
 3. Fill the model card,
 4. Build a demo on top of the model.
 
-**Let’s get started** 🚀 
 
-### **Model Training**
+## Important Dates
 
-You can find the notebook here and adjust it according to your own dataset 👇  
+TODO: Add official dates
+
+
+## Getting Started  🚀 
+
+To get started, join us in [hf.co/join/discord](http://hf.co/join/discord) and take the role #open-source, and meet us in #keras-working-group channel. 
+
+We will be hosting our demos in this organization on Hugging Face Hub: [keras-dreambooth](https://huggingface.co/keras-dreambooth), send a request to join [here](https://huggingface.co/organizations/keras-dreambooth/share/RMocthadPgpxxUDHtAesrbBzieDLgUfPmv) if you’d like to make a submission 🙂
+
+## Submission Categories
```

Comment:
I think this doesn't make sense without seeing the whole workflow and what people will do, at this line of guide you don't know much about what you will do as a participant, so I'd rather keep it below.